pom.xml

```diff
@@ -3,17 +3,17 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.509</version>
+		<version>1.557</version>
```

Comment:
we could go higher than that. the reason for that is that the stapler has a NPE when run in java8 before that version